gammapy/irf/psf_table.py

```diff
@@ -598,31 +603,34 @@ def info(self):
         """Print basic info"""
         print(str(self))
 
-    def plot_psf_vs_rad(self, energies=[1e4, 1e5, 1e6]):
+    def plot_psf_vs_rad(self, energies=None, ax=None, **kwargs):
```

Comment:
Is  ```energy=None``` is really safe/meaningful? Isn't it better not to put any default?

Reply:
Mutable objects such as dicts, lists or "Quantity" should be avoided as default arguments, because they are created when the function definition is executed and so are shared between multiple instances. The patter we adopted in Gammapy is to put `None` instead and put the default in the first line of the function `if energies is None: ...`. I think it's nice to have a default, but I don't insist to keep it.